src/payment/strategies/adyenv2/adyenv2-payment-strategy.ts

```diff
@@ -87,7 +87,18 @@ export default class AdyenV2PaymentStrategy implements PaymentStrategy {
         return this._store.dispatch(this._orderActionCreator.submitOrder(order, options))
             .then(() => {
                 if (paymentData && isVaultedInstrument(paymentData)) {
-                    return this._store.dispatch(this._paymentActionCreator.submitPayment({...payment, paymentData}));
+                    const paymentPayload = {
+                        methodId: payment.methodId,
+                        gatewayId: payment.gatewayId,
+                        paymentData: {
+                            nonce:  JSON.stringify({
```

Comment:
Instead of passing everything in the nonce, we can now use the `formattedPayload` key and bigypay client will just send that payload to bigpay:

```ts
paymentData: {
	formattedPayload: {
		nonce: paymentData.nonce,
		browser_info: { 
 			user_agent,
            accept_header,
            language,
            color_depth,
            screen_height,
            screen_width,
            time_zone_offset,
            java_enabled,
		},
	}
}
```